cmd/brew-portable-package.rb

```diff
@@ -25,12 +25,11 @@
 ARGV.named.each do |name|
   name = "portable-#{name}" unless name.start_with? "portable-"
   begin
+    deps = Utils.popen_read("brew", "deps", "--include-build", name).split("\n")
+    safe_system "brew", "install", "-s", *deps
```

Comment:
This should be guarded to Linux-only I think.

Reply:
Done.